Pipfile

```diff
@@ -11,14 +11,13 @@ open-aea-cli-ipfs = "==1.20.0"
 
 [dev-packages]
 asn1crypto = "==1.4.0"
-atheris = "==2.0.11"
```

Comment:
Remove the `atheris` dependency because it is not supported on Windows.

The options would be to:
1. Add `atheris` in the `setup.py` with a condition to install only if not on Windows. This however would add an unnecessary dependency to our framework.
2. Replace `pipenv` with `poetry` which supports "environment markers" and install dev dependency only if not on Windows. We could do this if we reopen #93.
3. Remove the dependency and let the dev install it if needed. This does not cause any issues since we already have the following logic in the fuzzy tests:

```python
try:
    import atheris  # type: ignore
except (ImportError, ModuleNotFoundError):
    pytestmark = pytest.mark.skip
```
The best option, for now, would be (3.).